src/constants/analytics.ts

```diff
@@ -206,6 +208,15 @@ export type AnalyticsEventData<T extends AnalyticsEvent> =
         type: string;
         id: string;
       }
+    : T extends AnalyticsEvent.TransferNotification
+    ? {
+        type: TransferNotificationTypes | undefined;
+        toAmount: number | undefined;
+        timeSpent: Record<string, number> | number | undefined;
+        txHash: string;
+        status: string;
```

Comment:
nit: can be `status: "new" | "success" | "error"`

Reply:
oh good point, we should definitely use string literals here 👍